share/commands/agent-inventory

```diff
@@ -4,5 +4,24 @@
 # @man Even if the agent will do it regularly, it can be used to force the update
 # @man after a modification on the node.
 # @man This won't affect the node state, but only update server-side information.
+# @man +
+# @man *Options*:
+# @man +
+# @man *-v*: run the agent in verbose mode
+# @man +
+# @man *-d*: run the agent in debug mode
 
-/var/rudder/cfengine-community/bin/cf-agent -KC -D force_inventory
+VERBOSITY=""
+
+while getopts "vd" opt; do
+  case $opt in
+    v)
+      VERBOSITY="-v"
```

Comment:
The last option is kept, like CFEngine does. I added [-i|-v|-d] in the general description to explain these options are mutually-exclusive.